frontend/src/metabase/nav/containers/AppBar.styled.tsx

```diff
@@ -32,16 +33,31 @@ export const LogoIconWrapper = styled.div`
   }
 `;
 
-export const SidebarButton = styled(Icon)`
-  border: 1px solid ${color("border")};
-  padding: ${space(1)};
-  border-radius: 4px;
-  margin-left: ${space(1)};
-  color: ${color("text-medium")};
-
+const SidebarIcon = styled(Icon)`
   &:hover {
-    color: ${color("brand")};
-    border-color: ${color("brand")};
     cursor: pointer;
+    color: ${color("brand")};
   }
 `;
+
+const SidebarButtonRoot = styled.div`
+  margin-left: ${space(1)};
+  margin-top: ${space(1)};
+`;
+
+interface SidebarButtonProps {
+  isSidebarOpen: boolean;
+  onClick: () => void;
+}
+
+export function SidebarButton({ isSidebarOpen, onClick }: SidebarButtonProps) {
```

Comment:
nit: Code we move it to a separate file? I thought that the convention is to have only styled components in `.styled.tsx` files.